src/modules/Accounts/hooks/useAccounts/useAccountInfo.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import * as apiClient from 'utilities/api/lisk/apiClient';
+import { resetAccountSummary, setAccountSummary } from '../../store/actions';
+import { selectAccountSummary } from '../../store/selectors';
+import { useCurrentAccount } from './useCurrentAccount';
+
+// eslint-disable-next-line
+export function useAccountInfo() {
+  const [account] = useCurrentAccount();
+  const dispatch = useDispatch();
+
+  const address = account?.metadata?.address;
+
+  const getAccount = (add) => apiClient.apiClient.getAccount(add);
+
+  useEffect(() => {
+    if (address) {
```

Comment:
usually for improve readability this kind of condition community write an if with return state false
```suggestion
    if (!address) {
      return false
    }
```